bangc-ops/mlu_op.h

```diff
@@ -10849,6 +10849,237 @@ mluOpPointsInBoxes(mluOpHandle_t handle,
                    const void *boxes,
                    const mluOpTensorDescriptor_t points_indices_desc,
                    void *points_indices);
+// Group:RoiAlign
+/*!
+ * @brief Computes the gradients of images \b grads_image based on the gradients \b grads and
+ * bounding boxes \b boxes to perform the backpropagation of ::mluOpRoiAlign operation. To use
+ * maximum pooling mode or average pooling mode in this operation, call ::mluOpRoiAlignBackward_v2.
+ *
+ * @param[in] handle
+ * Input. Handle to an MLUOP context that is used to manage MLU devices and queues in
+ * the roi_align_backward operation. For detailed information, see ::mluOpHandle_t.
+ * @param[in] grads_desc
+ * Input. The descriptor of the gradient tensor in the backpropagation process. For detailed
+ * information, see ::mluOpTensorDescriptor_t.
+ * @param[in] grads
+ * Input. Pointer to the MLU memory that stores the gradient tensor.
+ * @param[in] boxes_desc
+ * Input. The descriptor of the bounding boxes tensor. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[in] boxes
+ * Input. Pointer to the MLU memory that stores the bounding boxes tensor.
+ * @param[in] spatial_scale
+ * Input. A scaling factor that specifies how to map the box coordinates in the origin image to
+ * the coordinates in the output.
+ * @param[in] sampling_ratio
+ * Input. The number of sampling points in the grid used to compute the output.
+ * @param[in] aligned
+ * Input. A boolean value which determines whether to shift the boxes by 0.5 pixel.
+ * @param[in] grads_image_desc
+ * Input. The descriptor of the gradients tensor of the original images.
+ * @param[out] grads_image
+ * Output. Pointer to the MLU memory that stores the gradients tensor of the original images.
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM, ::MLUOP_STATUS_ARCH_MISMATCH.
+ *
+ * @par Data Type
+ * - This function supports the following data types for gradient tensor \b grads, boxes tensor
+ *   \b boxes, and output tensor \b grads_image. Data type of all tensors should be the same.
```

Comment:
- The data types of all tensors should be the same.
- The supported data types of input and output tensors are as follows: